### PR TITLE
Update version upgrade message to include brew

### PR DESCRIPTION
**Pull request by jstvz**
@davisagli Should we attempt to detect whether cci has been installed via `brew` to display the appropriate upgrade command?

# Critical Changes

# Changes

# Issues Closed

### CODE REVIEW

**Comment by davisagli** [comment]
@jstvz that would be neat. Just based on the path of the cumulusci module?

**Comment by jstvz** [comment]
Yeah, can look for `Cellar` for darwin... linuxbrew might be trickier. I'll look it up.

File: Makefile [hunk 1]
```diff
@@ -60,7 +60,7 @@ test-all: ## run tests on every Python version with tox
 	tox
 
 coverage: ## check code coverage quickly with the default Python
-	coverage run --source cumulusci pytest
```
Comment (davisagli) [inline Makefile]: Oh nice.

File: cumulusci/cli/cci.py [hunk 2]
```diff
@@ -40,7 +40,7 @@
 from cumulusci.core.exceptions import TaskNotFoundError
 from cumulusci.core.utils import import_class
 from cumulusci.cli.config import CliRuntime
-from cumulusci.cli.config import get_installed_version
+from cumulusci.cli.config import get_installed_version, get_cci_upgrade_command
```
Comment (davisagli) [inline cumulusci/cli/cci.py]: This should be imported from its real location in cumulusci.utils

**Pull request merged**